### ISSUE TITLE
Support TMPDIR [was: Build failing on ubuntu 20.04]

### ISSUE THREAD

**Issue by lbolla** [molior-dbs/molior-tools#49]
When running `molior-deploy` on Ubuntu 20.04, I get various errors all complaning about `/dev/null` being readonly. E.g.:
```
/usr/bin/apt-key: 732: /usr/bin/apt-key: cannot create /dev/null: Permission denied
```

This doesn't happen on older Ubuntus.

**Comment by lbolla** [comment]
Upon further investigation, the issue seems to be caused by /tmp/ being mounted with "nodev,noexec" flags. It has nothing to do with Ubuntu version.

**Comment by lbolla** [comment]
Closing, as it's not an issue, except maybe it'd be nice of molior-tools checked if `/tmp` is mounted correctly before starting to build.

**Comment by randombenj** [comment]
We should support `TMPDIR` env variable dough.

**Comment by dario-gallucci** [comment]
Fixed in 08c30ff29725aef3dfcf87a81b5f4ebbf0e1e96a
